LoRa.B827EBFFFED61EBD.json

```diff
@@ -0,0 +1,18 @@
+{ 
+ "gateway_conf":{ 
+     "gateway _ID":"B827EBFFFED61EBD", 
+     "servers":[ 
+          { 
+          "server_address":"router.eu.thethings.network", 
+          "serv_port_up"1700, 
```

Comment:
The red here indicates it's invalid json. It's missing the colon between the key and the value.